libs/datasets/sources/can_scraper_helpers.py

```diff
@@ -262,7 +262,7 @@ def check_variable_coverage(self, variables: List[ScraperVariable]):
         provider_mask = self.all_df[Fields.PROVIDER] == provider_name
         counts = self.all_df.loc[provider_mask, Fields.VARIABLE_NAME].value_counts()
         variables_by_name = {var.variable_name: var for var in variables}
-        for variable_name, count in counts.iteritems():
+        for variable_name, count in counts.items():
```

Comment:
[`pandas.DataFrame.iteritems()`](https://pandas.pydata.org/pandas-docs/version/1.5/reference/api/pandas.DataFrame.iteritems.html#pandas.DataFrame.iteritems) is deprecated in favor or `...items()`